Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.7-alpine
+WORKDIR /app
+COPY . ./
+RUN apk update
+RUN apk add --no-cache build-base gcc cmake git
+RUN gem update --system && gem update bundler && gem install bundler jekyll:3.9.3
+RUN echo "$PWD"
```

Comment:
I'd comment this out or remove it as it'll hinder layer reuse.

Reply:
I'll give it a shot.  TY.

Reply:
> I'd comment this out or remove it as it'll hinder layer reuse.

Specifically the `RUN apk add --no-cache build-base gcc cmake git` line?

Reply:
@sixlettervariables Sorry, I'm not sure which line would hinder layer reuse.  It looks like both the `apk` lines and the `gem update` line are required.  I can remove the echo line, though.

Reply:
Each line is a new layer, so order matters. If layer 1 changes, all subsequent layers change.

The trick is to order lines based on slowest changing to fastest changing.

The lines you don't need are the echo PWD and the line checking the Jekyll version.